rest/views.py

```diff
@@ -89,8 +90,15 @@ class VocabularyList(ListCreateAPIView):
 
     def get_queryset(self):
         """Filter to vocabularies for a repository"""
-        return Repository.objects.get(
-            slug=self.kwargs['repo_slug']).vocabulary_set.order_by('id')
+        queryset = Vocabulary.objects.filter(
+            repository__slug=self.kwargs['repo_slug']
+        )
+        learning_resource_type = self.request.query_params.get(
+            'learning_resource_type', None)
+        if learning_resource_type is not None:
+            queryset = queryset.filter(
+                learning_resource_types__name=learning_resource_type)
+        return queryset.order_by('id')
```

Comment:
If we don't order it results may be in a random order. It may not matter though, I was erring on the side of caution here